free-1.md

```diff
@@ -58,11 +74,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I feel like this language would require a lot of work in both domains. The work would require code that is more key word/ natural langauge based which would be alot in themsleves. additionally it would require that each action of a keyword be brought to life through automated functionality within the domain of database set up. 
+
```

Comment:
Agreed. It definitely has interesting language design questions, but a lot of implementation challenges too.